addons/mail/static/src/discuss/core/public/welcome_page.js

```diff
@@ -22,6 +23,12 @@ export class WelcomePage extends Component {
         });
         this.audioRef = useRef("audio");
         this.videoRef = useRef("video");
+        onWillStart(() => {
+            if (this.props.data.channelData.defaultDisplayMode === "video_full_screen") {
+                this.enableMicrophone();
+                this.enableVideo();
+            }
+        });
```

Comment:
`enableMicrophone/Video` requires ref. `onWillStart()` happens before DOM render.
Likely need `onMounted()` instead?

_(also `onWillStart` from experience is prone to problems that requires replacing it, so I'd strongly advice to avoid it. I even heard some rumors that there were talk to even consider removing it!)_

Reply:
Ah I see, it with willStart but I guess that it's because I don't wait for slow promises, I change that